apps/frontend/src/modules/auth/auth-api.ts

```diff
@@ -36,6 +38,36 @@ class AuthApi extends BaseHTTPApi {
 		return await response.json<UserDto>();
 	}
 
+	public async requestResetPassword(payload: EmailDto): Promise<boolean> {
+		const response = await this.load(
+			this.getFullEndpoint(AuthApiPath.FORGOT_PASSWORD, {}),
+			{
+				contentType: ContentType.JSON,
+				hasAuth: false,
+				method: "POST",
+				payload: JSON.stringify(payload),
+			},
+		);
+
+		return await response.json<boolean>();
```

Comment:
Is this valid?

Reply:
If you receive a boolean from Backend, then yes